src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -259,3 +259,9 @@
 (defn bridge-disabled?
   [token-symbol]
   (not (constants/bridge-assets token-symbol)))
+
+(defn signature-rsv
+  [signature]
+  {:r (subs signature 0 64)
+   :s (subs signature 64 128)
+   :v (subs signature 128 130)})
```

Comment:
Duplicated functions here?

https://github.com/status-im/status-mobile/pull/21589/files#r1831380457

Reply:
Oh, I see you are removing that file in this PR. Better to coordinate with @clauxx the path forward for this so we don't waste QA time testing the same code twice (also solving rebase conflicts better).

As I mentioned in Cristian's PR, would love to see some unit tests for this function, so it can serve as documentation for this sensible wallet code.

Reply:
Nope. I removed that one.
This one is fixed + it returns only the signature, and not the vector containing the tx-hash (which is kinda dumb in the original one).